functions/uploads.py

```diff
@@ -97,19 +97,17 @@ def ingest_upload(event: dict, context: BackgroundContext):
         # NOTE: this needs to happen after TrialMetadata.patch_assays
         # in order to avoid violating a foreign-key constraint on the trial_id
         # in the event that this is the first upload for a trial.
-        downloadable_file_ids = []
         for artifact_metadata, additional_metadata in downloadable_files:
             print(f"Saving metadata to downloadable_files table: {artifact_metadata}")
             with saved_failure_status(job, session):
-                df = DownloadableFiles.create_from_metadata(
+                DownloadableFiles.create_from_metadata(
                     trial_id,
                     job.assay_type,
                     artifact_metadata,
                     additional_metadata=additional_metadata,
                     session=session,
                     commit=False,
                 )
-                downloadable_file_ids.append(df.id)
```

Comment:
`df.id == None` always, because `df` hasn't actually been committed to the database yet.